src/Umbraco.Web.UI.Client/src/packages/core/content/workspace/views/edit/content-editor.element.ts

```diff
@@ -78,6 +81,16 @@ export class UmbContentWorkspaceViewEditElement extends UmbLitElement implements
 		if (!this._tabs || !this.#structureManager) return;
 		const routes: UmbRoute[] = [];
 
+		if (this._hasRootGroups) {
+			routes.push({
+				path: `tab/generic`,
```

Comment:
in the Content Type Designer, we use the path `root` to avoid eventual collision with other tabs like `tab/generic`.
I dont think the merge feature supports merging root groups with a tab of this name. And the Merge functionality does no longer exist on the server, so I do not think that the old way needs to be respected any longer.

Since you read the docs, then maybe there is a problem in this new approach where they cannot merge? How do you see that?

Otherwise I would prefer looking to solve it similar to the Content Type Designer, as seen in `src/packages/core/content-type/workspace/views/design/content-type-design-editor.element.ts`

Reply:
`root` looks like it'll be just as fine to me.  I think I added `generic` to the route just to match the name of the tab.  Feel free to adjust this as you see fit.

Reply:
Cool, thanks